README.md

```diff
@@ -126,15 +126,17 @@ evm:
     keyring-pass-env-name: ETH_PASSWORD
     signing-key: ${ETH_SIGNING_KEY}
     keyring-dir: ~/.pigeon/keys/evm/eth-main
-    gas-adjustment: 1.2
+    gas-adjustment: 1.6
+    tx-type: 2
```

Comment:
And having read through this, I don't know what `tx-type` is supposed to represent. Would it be wrong to switch on e.g. `chain-id` instead and remove this new field?

Reply:
In EIP-1559, legacy transaction was defined "Type 0x00" and new kind of transaction was defined "Type 0x02".
Go ethereum library already provides Type() as 0 and 2(uint8)
https://eips.ethereum.org/EIPS/eip-1559

Reply:
I was going to remove gas-adjustment field in config and use hardcoded 1.6 or 2.0. But the config might be better.

Reply:
> In EIP-1559, legacy transaction was defined "Type 0x00" and new kind of transaction was defined "Type 0x02".
> Go ethereum library already provides Type() as 0 and 2(uint8)
> https://eips.ethereum.org/EIPS/eip-1559

This would go great in a comment on the field.

Important to check here: if I leave out the `tx-type` field, do I get an error on startup or does it just default to `0`? Because one of those is very bad.

Re: config vs hardcode. Is it wrong for the user to put different values in the config here? If it is we should just hardcode the value.

Reply:
Other fields don't have comments. I don't think we need comments only for this field update.